### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by capitanbatata**
Created a template on how submit new issue. The idea is to force us and the community to think about what is desired, and also to have meaningful discussions about the possible ways of solving the problem.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+This is an issue template. 
+
+If submitting a **bug report** please specify:
+
+- Expected behavior
+- Actual behavior
+- Steps to reproduce the actual behavior
+
```
Comment (pjljvandelaar) [inline ISSUE_TEMPLATE.md]: Please include the trace which can be obtained using the TorXakis command `trace $> filename`
Please add the SMT log file which can be obtained by starting `txsserver` with the flag `--smt-log`.

Comment (keremispirli) [reply ISSUE_TEMPLATE.md]: OS can also be useful

Comment (pjljvandelaar) [reply ISSUE_TEMPLATE.md]: see my review remarks ;-)

Comment (keremispirli) [reply ISSUE_TEMPLATE.md]: ah :)

**Comment by pjljvandelaar** [comment]
See e.g. stack: https://github.com/commercialhaskell/stack/issues/new
Entering a new issues results in the following text:
```
Do you have a question regarding stack's usage that isn't covered by [the docs](http://haskellstack.org)?
In that case please ask your question on [Stack Overflow](http://stackoverflow.com) and use [the haskell-stack tag](http://stackoverflow.com/questions/tagged/haskell-stack).
This way your question will be more easily discoverable by other people with the same question.

Question related to stack project templates? Please report it at the [stack-templates](https://github.com/commercialhaskell/stack-templates) repository instead.

If you're reporting a bug please follow the steps below:

Make sure that you are using the latest release (currently stack-1.6.1).
See the [upgrade instructions](http://docs.haskellstack.org/en/stable/install_and_upgrade/#upgrade) to upgrade.

Please use the following schema for your bug report:

### General summary/comments (optional)

### Steps to reproduce

For example:

1. Remove directory *blah*.
2. Run command `stack blah`.
3. Edit file blah.
4. Run command `stack blah`.

Include any `.yaml` configuration if relevant.

### Expected

What you expected to see and happen.

### Actual

What actually happened.

If you suspect that a stack command misbehaved, please include the output of that command in `--verbose` mode.
If the output is larger than a page please paste the output in a [Gist](https://gist.github.com/).

$ stack <your command here> <args> --verbose
<output>

### Stack version

$ stack --version
Version 1.5.0, Git revision 63267f94d7c819cbecc2d59aa259d17240838e43 (4845 commits) x86_64 hpack-0.17.1

### Method of installation

* Official binary, downloaded from stackage.org or fpcomplete's package repository
* Via cabal-install
* An unofficial package repository (please specify which)
* Other (please specify)
```

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (capitanbatata) [reply ISSUE_TEMPLATE.md]: I still don't know if we will support a `trace` command in that form, so I'm only mentioning "include a TorXakis trace". Also the `txsserver` might be scrapped away, so I'm only adding "If applicable, SMT log file."

**Comment by capitanbatata** [comment]
Changes are incorporated. See what do you think about this new version.

**Pull request merged**